atom/browser/ui/cocoa/atom_touch_bar.mm

```diff
@@ -467,4 +494,120 @@ - (void)updateSegmentedControl:(NSCustomTouchBarItem*)item
     control.selectedSegment = selectedIndex;
 }
 
+- (NSTouchBarItem*)makeScrubberForID:(NSString*)id
+                     withIdentifier:(NSString*)identifier {
+  std::string s_id([id UTF8String]);
+  if (![self hasItemWithID:s_id]) return nil;
+
+  mate::PersistentDictionary settings = settings_[s_id];
+  base::scoped_nsobject<NSCustomTouchBarItem> item([[NSClassFromString(
+      @"NSCustomTouchBarItem") alloc] initWithIdentifier:identifier]);
+
+  int width = 320;
+  int height = 30;
+  settings.Get("frameWidth", &width);
+  settings.Get("frameHeight", &height);
```

Comment:
I left these in because I didn't know what they actually did.  AFAICS changing these values affects nothing so maybe removing them is the better option.  Do you know if these affect anything?